editor/js/Sidebar.Scene.js

```diff
@@ -181,7 +181,23 @@ function SidebarScene( editor ) {
 	backgroundEquirectangularTexture.setDisplay( 'none' );
 	backgroundRow.add( backgroundEquirectangularTexture );
 
+	const backgroundColorSpaceRow = new UIRow();
+	backgroundColorSpaceRow.setDisplay( 'none' );
+	backgroundColorSpaceRow.setMarginLeft( '120px' );
+
+	const backgroundColorSpace = new UISelect().setOptions( {
+
+		'': 'No Color Space',
+		'srgb-linear': 'sRGB (linear)',
+		'srgb': 'sRGB',
+
+	} ).setWidth( '150px' );
+	backgroundColorSpace.setValue( THREE.NoColorSpace );
```

Comment:
Possible to swap the default based on 8-bit or >=16-bit textures here?

Reply:
At that point in code there is no access to the texture yet.

Further down in `refreshUI()` the defined texture is evaluated:

```js
backgroundColorSpace.setValue( scene.background.colorSpace );
```
I don't think it's possible to suggested a color space value here since `refreshUI()` is also executed when the editor is reloaded and the serialized texture is restored.

A color space must be set earlier when the texture is actually loaded.

Reply:
I see, ok!